.pre-commit-config.yaml

```diff
@@ -43,6 +43,7 @@ repos:
         # AL07: Avoid table aliases in from clauses and join conditions <- we sometimes do this for long names
         # LT05: Line is too long <- not useful. mostly triggers for comments
         # LT12: Files must end with a single trailing newline <- already done by trailing-whitespace
+        # If you update this rule, please also update the rules in justfile, the fmt-sql recipe
```

Comment:
Or maybe use a same config file to avoid this

Reply:
That would only work if pre-commit were not pretty stubborn in not using it.

Reply:
Guess it's enough with this comment.

Reply:
We will know it's not enough when we konw loll